examples/typescript/src/App.tsx

```diff
@@ -20,7 +20,7 @@ const client = new GraphQLClient({
 
 export const allPostsQuery = `
   query {
-    allPosts(orderBy: createdAt_DESC, first: 20) {
```

Comment:
did you remove the order and limit because they're not needed, or?

Reply:
I removed them because they are not used by the example and they break if used with the included `json-graphql-server` which doesn't implement those parameters.

Reply:
I thought we were already using json-graphql-server, didn't it work?

Reply:
This example was broken when using the local json-graphql-server for this reason, but working when pointing to the remote server.